wolox-books/src/app/app-routing.module.ts

```diff
@@ -1,8 +1,13 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { RegisterComponent } from './screens/Signup/components/register/register.component';
+import { LoginComponent } from './screens/Login/components/login/login.component';
```

Comment:
The way you should organize this is:
```
screens/
|- login/
    |- components/
    |- login.component.ts
    |- login.component.html
    |- login.component.scss
|- signup/
    |- components/
    |- signup.component.ts
    |- signup.component.html
    |- signup.component.scss
```
This way, the screen components are at the screen level folder and the components are only for the generic components of the screen